docs/extend/modular-resources/examples/csi.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Add a CSI Camera as a Modular Resource"
+linkTitle: "CSI Camera"
+weight: 40
+type: "docs"
+description: "How to add a CSI Camera as a modular resource of your robot."
```

Comment:
```suggestion
description: "Use the viam:camera:csi model to add a CSI Camera to your robot."
```

This is the text that shows up in cards and previews - I think we want to avoid startign that with "How to"

Reply:
Sounds good!